src/utils/fiatToCrypto.js

```diff
@@ -23,20 +23,31 @@ import { getEnv } from 'configs/envConfig';
 
 import type { AltalixTrxParams, SendwyreRates, SendwyreTrxParams } from 'models/FiatToCryptoProviders';
 import type SDKWrapper from 'services/api';
+import { RAMP_CURRENCY_TOKENS } from '../constants/assetsConstants';
 
-export function rampWidgetUrl(address: string, email?: string, plrMode?: boolean) {
+const PILLAR = 'Pillar';
+
+export function rampWidgetUrl(
+  address: string,
+  email?: string,
+  fiatCurrency: string,
+  fiatValue: string,
+) {
+  const values = RAMP_CURRENCY_TOKENS.join(',');
```

Comment:
Could we change the variable name `values` to something more specific please? `values` is way too generic!

Reply:
Done